frontend/src/app/spot/styles/sass/components/modal-overrides.sass

```diff
@@ -9,14 +9,13 @@
 .modal-delivery-element
   display: none
 
-@media #{$spot-mq-mobile}
-  .spot-modal.op-file-picker
-    min-height: calc(100vh - 5rem)
+.spot-modal.op-file-picker
+  .op-file-picker--scrollable-content
```

Comment:
This can just be done in the ``file picker styles I htink

Reply:
just with the view on the location picker, shouldn't we have this style overwrites here, instead of for both, file and location picker locally?

Reply:
moved to own class